### PR TITLE
OperationDefinition for generating DocRef

### PR DESCRIPTION
**Pull request by omarisgreat**

### CODE REVIEW

File: OperationDefinition/OperationDefinition-genomics-generate.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  "id": "OperationDefinition-genomics-generate",
+  "url": "https://fhir.nhs.uk/OperationDefinition/genomics-generate",
+  "version": "0.0.1",
+  "name": "GenomicsGenerate",
+  "status": "draft",
+  "kind": "operation",
+  "experimental": false,
+  "date": "2024-09-04",
+  "publisher": "NHS England",
+  "description": "Used to generate a DocumentReference resource for a genomic data file, optionally pulling the referenced data file to a central data store and generating a DRS url for the data file.\r\nCustomised backport of the R5 DocumentReference-generate OperationDefinition: http://hl7.org/fhir/OperationDefinition/DocumentReference-generate.\r\nSpecific profiles for input or output parameters, e.g. for certain data file types, are pending further investigation by GEL.",
```
Comment (KazeemHamzat) [inline OperationDefinition/OperationDefinition-genomics-generate.json:12]: description: May be start with: An operation used generate a DocumentReference resource for..............................

**Review by KazeemHamzat — APPROVED** [review]
Added a likely suggestion to start the description content

File: OperationDefinition/OperationDefinition-genomics-generate.json (hunk 1 above)
Comment (omarisgreat) [reply OperationDefinition/OperationDefinition-genomics-generate.json]: Added

**Comment by omarisgreat** [comment]
Apologies, added another resource (input for operation)

**Review by KazeemHamzat — APPROVED** [review]
All looks good.

**Pull request merged**